pkg/docker/util_test.go

```diff
@@ -78,6 +78,32 @@ func TestCheckAllowedUser(t *testing.T) {
 			onbuild:     []string{"USER 501", "VOLUME /data"},
 			expectErr:   false,
 		},
+		{
+			name:        "AllowedUIDs is set, numeric user with group",
+			allowedUIDs: rangeList("1-"),
+			user:        "5:5000",
+			expectErr:   false,
```

Comment:
Should add a test for:

```
5:users
```

That is, named group if want to ensure coverage of all cases.